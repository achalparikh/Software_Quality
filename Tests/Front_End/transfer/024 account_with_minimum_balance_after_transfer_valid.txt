login
standard
ACHAL PARIKH
transfer
00001
00011
1000
logout
quit