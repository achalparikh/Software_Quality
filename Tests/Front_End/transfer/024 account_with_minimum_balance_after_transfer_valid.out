Enter user type - "standard" or "admin":  
Logging in as standard user - enter name: 
You are now logged in as "ACHAL PARIKH" 
COMMAND transfer - enter FROM account number: 
Transferring from 00001 - enter TO account number: 
Transferring to 00011 - enter amount:
Transfer of 1000.00 - complete!
ALERT: Balance is zero
You are now logged out
